### ISSUE TITLE
Feature request: Follow up from #169, respecting cocoa keystrokes like safari tab-switching

### ISSUE THREAD

**Issue by arubdesu** [ProfileCreator/ProfileCreator#174]
As I was hoping that all my cert payloads would show up (I didn't think to scroll) I tried tab-switching (command-shift-left or right bracket) and couldn't get it to 'page' through the payloads like you would through e.g. Safari or Finder tabs. Does that seem do-able/worthwhile?

**Comment by apizz** [comment]
@arubdesu I agree that some shortcut would be helpful.  I live in Chromium world, so I do a lot of Command-Shift-left/right arrow to switch tabs.

Although, we don't currently have anyone who's taken up the project ...
